hipogram/posts/admin.py

```diff
@@ -1,5 +1,18 @@
 from django.contrib import admin
+from .models import Post, Tag
 
-from .models import Post
+admin.site.register(Tag)
 
-admin.site.register(Post)
+
+class PostAdmin(admin.ModelAdmin):
+    list_display = ("created_by", "creation_datetime", "Tags")
+    list_filter = ("created_by", "tags")
+
+    def Tags(self, obj):
```

Comment:
I prefer to use the name of the model -which is `post` here- instead of `obj`.

Reply:
> Lowercase name would be better.

When I used lowercase i.e. 'tags', it didn't work. I think this is because 'tags' is also a field name in the Post model.

Reply:
> I think this is because 'tags' is also a field name in the Post model.


Make sense. I prefer to use `def _tags(...` in these situations, not sure if it's best practice.